mod-orders/mod-orders.postman_collection.json

```diff
@@ -0,0 +1,901 @@
+							],
+							"port": "{{okapiport}}",
+							"path": [
+								"_",
+								"proxy",
+								"modules"
+							]
+						},
+						"description": "GET /_/proxy/modules requests that returns 200"
+					},
+					"response": []
+				},
+				{
+					"name": "GET /orders - get list of orders",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "eb2052a4-e388-4860-80be-ea96976bcf21",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "3bf00f6d-8798-4764-a256-2d360ea02876",
+								"exec": [
+									"var jsonData = pm.response.json();",
+									" ",
+									"pm.test(\"Status code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"pm.test(\"Multiple orders exist\", function () {",
+									"    pm.expect(jsonData.composite_purchase_orders).to.have.length > 1;",
+									"});",
+									"",
+									"pm.test(\"Each order has required fields\", function(){",
+									"    pm.expect(jsonData.composite_purchase_orders[0].id).to.exist;",
+									"    pm.expect(jsonData.composite_purchase_orders[0].notes).to.exist;",
+									"    pm.expect(jsonData.composite_purchase_orders[0].po_number).to.exist;",
+									"    pm.expect(jsonData.composite_purchase_orders[0].po_lines).to.exist;",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "X-Okapi-Tenant",
+								"value": "{{xokapitenant}}"
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json"
+							},
+							{
+								"key": "x-okapi-token",
+								"value": "{{xokapitoken}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{protocol}}://{{url}}:{{okapiport}}/orders",
+							"protocol": "{{protocol}}",
+							"host": [
+								"{{url}}"
+							],
+							"port": "{{okapiport}}",
+							"path": [
+								"orders"
+							]
+						},
+						"description": "GET /orders/ requests that returns 200"
+					},
+					"response": []
+				},
+				{
+					"name": "POST /orders - create empty order",
+					"event": [
+						{
+							"listen": "prerequest",
+							"script": {
+								"id": "eb2052a4-e388-4860-80be-ea96976bcf21",
+								"exec": [
+									""
+								],
+								"type": "text/javascript"
+							}
+						},
+						{
+							"listen": "test",
+							"script": {
+								"id": "3bf00f6d-8798-4764-a256-2d360ea02876",
+								"exec": [
+									"var jsonData = pm.response.json();",
+									" ",
+									"pm.test(\"Status code is 201\", function () {",
+									"    pm.response.to.have.status(201);",
+									"});",
+									"",
+									"pm.test(\"Empty list of notes exist\", function () {",
```

Comment:
Is the API supposed to return empty notes even if we supplied notes while placing an order? or is it that the functionality is not yet implemented to return notes?

Reply:
Yes, if it is an empty order then API will return empty notes and empty po_lines.